ui/components/app/modals/contract-details-modal/contract-details-modal.js

```diff
@@ -253,53 +253,53 @@ export default function ContractDetailsModal({
-                title={
-                  copiedToAddress
+            <Tooltip
+              position="top"
+              title={
+                copiedToAddress ? t('copiedExclamation') : t('copyToClipboard')
+              }
+            >
+              <ButtonIcon
+                display={DISPLAY.FLEX}
+                iconName={
+                  copiedToAddress ? ICON_NAMES.COPY_SUCCESS : ICON_NAMES.COPY
+                }
+                onClick={() => handleCopyToAddress(toAddress)}
+                color={Color.iconMuted}
+                ariaLabel={
+                  copiedTokenAddress
                     ? t('copiedExclamation')
                     : t('copyToClipboard')
                 }
-              >
-                <ButtonIcon
-                  iconName={
-                    copiedToAddress ? ICON_NAMES.COPY_SUCCESS : ICON_NAMES.COPY
-                  }
-                  onClick={() => handleCopyToAddress(toAddress)}
-                  color={Color.iconMuted}
-                />
-              </Tooltip>
-            </Box>
-            <Box marginTop={5} marginRight={5}>
-              <Tooltip position="top" title={t('openInBlockExplorer')}>
-                <Button
-                  className="contract-details-modal__content__contract__buttons__block-explorer"
-                  type="link"
-                  onClick={() => {
-                    const blockExplorerTokenLink = getAccountLink(
-                      toAddress,
-                      chainId,
-                      {
-                        blockExplorerUrl: rpcPrefs?.blockExplorerUrl ?? null,
-                      },
-                      null,
-                    );
-                    global.platform.openTab({
-                      url: blockExplorerTokenLink,
-                    });
-                  }}
-                >
-                  <IconBlockExplorer
-                    size={16}
-                    color="var(--color-icon-muted)"
-                  />
-                </Button>
-              </Tooltip>
-            </Box>
+              />
+            </Tooltip>
+            <Tooltip position="top" title={t('openInBlockExplorer')}>
+              <ButtonIcon
+                display={DISPLAY.FLEX}
+                iconName={ICON_NAMES.EXPORT}
+                color={Color.iconMuted}
+                onClick={() => {
+                  const blockExplorerTokenLink = getAccountLink(
+                    toAddress,
+                    chainId,
+                    {
+                      blockExplorerUrl: rpcPrefs?.blockExplorerUrl ?? null,
+                    },
+                    null,
+                  );
+                  global.platform.openTab({
+                    url: blockExplorerTokenLink,
+                  });
+                }}
+                ariaLabel={t('openInBlockExplorer')}
+              />
+            </Tooltip>
           </Box>
         </Box>
       </Box>
```

Comment:
These are my updates 

### Before (Before David's updates as well so completely before this PR not taking all the credit)

<img width="1440" alt="Screenshot 2023-02-08 at 12 11 35 PM" src="https://user-images.githubusercontent.com/8112138/217644390-93a096a8-9f4a-4fca-b14c-1abffedc3fcb.png">

### After (Just to make sure I'm not breaking anything in this PR)

<img width="752" alt="Screenshot 2023-02-08 at 12 25 22 PM" src="https://user-images.githubusercontent.com/8112138/217644584-a39cda15-a6c2-4dc5-b0d8-5713d7868dec.png">

Also fixing some minor tooltip alignment 
Before
<img width="198" alt="Screenshot 2023-02-08 at 12 24 15 PM" src="https://user-images.githubusercontent.com/8112138/217644624-9ad935e6-e016-4bf0-ad66-68abebe46d7c.png">
After
<img width="189" alt="Screenshot 2023-02-08 at 12 25 04 PM" src="https://user-images.githubusercontent.com/8112138/217644625-69035608-f1f6-4786-a63d-7a190540fc72.png">